kube/.drone.yml

```
@@ -0,0 +1,58 @@
+---
+kind: pipeline
+type: kubernetes
+name: validate k8
```

it'd be k9, surely 🐩 
```suggestion
name: validate k8s
```